qtconsole/qtconsoleapp.py

```diff
@@ -415,7 +415,7 @@ def _init_asyncio_patch(self):
     def initialize(self, argv=None):
         # Fixes launching issues with Big Sur
         # https://bugreports.qt.io/browse/QTBUG-87014, fixed in qt 5.15.2
-        if sys.platform == 'darwin':
+        if sys.platform == 'darwin' and not QT6:
```

Comment:
I don't think this is the right solution @psobolewskiPhD because we'd probably have similar checks to thes one below in the future.

Therefore, I propose to use instead the `parse` function from the `packaging` module instead, as it was used in our tests until PR #570:

https://github.com/jupyter/qtconsole/blob/a2aa5a11c883f5fd648eb56c19bf463d1cad99b1/qtconsole/tests/test_jupyter_widget.py#L77

Let us know if you feel comfortable doing this change or you prefer for me to do it.

Reply:
Hi @ccordoba12 do you mean in the lines below, replace `QtCore.QVersionNumber.fromString` with `parse`? Rather than just skipping that QT5 code on QT6 that is.

Reply:
Yep, that's exactly what I meant.

Reply:
Looking at my comment in the issue I had thought about doing that, but then did the lazy way 🤣 
Anyhow, I think I've got it sorted—works for me locally with pyside 6.4.2